ansible/configs/openshift-demos/msa-cicd-eap-full.yml

```diff
@@ -61,3 +61,58 @@
         gogs_hostname: "gogs-{{ project_cicd }}.{{ apps_hostname_suffix }}"
         prune_deployments_selector_prod: ""
         prune_deployments_selector_stage: "comp-required!=true,app!=inventory"
+
+    # verify deployments in ci/cd project
+    - name: wait for deployments in project {{ project_cicd }} to complete (succeed or fail)
+      shell: "oc get pods -n {{ project_cicd }} | grep '\\-deploy' | grep 'Running'"
+      register: deployment_running
+      until: deployment_running|failed
```

Comment:
This would then be something like

```
until: deployment_running|from_json|json_query(...) )  is empty
```

determining '(...)' is the hard part. I think `.spec.name == 'deployment'  && .status.phase == 'Running'` for '-deploy'  pods should do it.